consul_exporter.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"flag"
+	_ "fmt"
```

Comment:
Hum, lots of unused imports that can be removed?

But I'd recommend adding:

    _ "net/http/pprof"

...so that it's easier to debug in production.